includes/class-wc-google-gtag-js.php

```diff
@@ -94,7 +94,7 @@ private function register_scripts(): void {
 					function %2$s(){dataLayer.push(arguments);}
 					// Set up default consent state.
 					for ( const mode of %4$s || [] ) {
-						%2$s( "consent", "default", mode );
+						%2$s( "consent", "default", { ...mode, "wait_for_update": 500 } );
```

Comment:
:question: 
How about letting merchants customize their own `wait_for_update` value?
```suggestion
						%2$s( "consent", "default", { "wait_for_update": 500, ...mode } );
```

Reply:
Nice suggestion! 4b9cc2c